qa/rpc-tests/fundrawtransaction.py

```diff
@@ -521,7 +521,7 @@ def run_test(self):
         #############################################
 
         #again, empty node1, send some small coins from node0 to node1
-        self.nodes[1].sendtoaddress(self.nodes[0].getnewaddress(), self.nodes[1].getbalance(), "", "", True)
+        self.nodes[1].sendtoaddress(self.nodes[0].getnewaddress(), self.nodes[1].getbalance("*", 0), "", "", True)
```

Comment:
This change should not be necessary if we're keeping 0 as the default for `minconf`.